tools/publishing/compute_role_spectrum.py

```
@@ -0,0 +1,369 @@
+        ctx.check_hostname = False
+        ctx.verify_mode = ssl.CERT_NONE
+        url = (
+            f"{SUPABASE_URL}/rest/v1/profiles"
+            f"?full_name=ilike.*{urllib.parse.quote(name)}*"
+            f"&select=id,full_name,role,email"
+        )
+        req = urllib.request.Request(url, headers={
+            'apikey': SUPABASE_KEY,
+            'Authorization': f'Bearer {SUPABASE_KEY}',
+            'Content-Type': 'application/json',
+        })
```

Ruff (0.15.21)

[error] Audit URL open for permitted schemes. Allowing use of `file:` or custom schemes is often unexpected.

(S310)